backend/index.js

```diff
@@ -114,28 +22,50 @@ const startServer = async () => {
         res.send('game selection');
     });
 
-    app.get('/patchnotes', async (req, res) => {
+    app.get('/patchnotes/overwatch', async (req, res) => {
         try {
-            const urls = await getPatchNotesUrls();
+            const urls = await urls_OW.getPatchNotesUrls_OW(OVERWATCH_URL);
             res.json(urls);
         } catch (error) {
-            console.error('Error in /patchnotes route:', error.message);
+            console.error('Error in /patchnotes/overwatch route:', error.message);
             res.status(500).send({ error: 'Error while fetching patch notes' });
         }
     });
 
-    app.get('/patchnotes/:year/:month', async (req, res) => {
+    app.get('/patchnotes/valorant', async (req, res) => {
+        try {
+            const urls = await urls_VAL.getPatchNotesUrls_VAL(VALORANT_URL);
```

Comment:
architecturally, the most fragile part of our application is the web scraping logic. The next time that Riot or Valve updates their website with a different layout or a different className, our scraping logic will break.

I mentioned this in your design document as well, but I think it will be much better if we can ingest the site into our own database, so that we can store a "snapshot" of the data when the logic is working.

You can add a new (private? internal? otherwise secured?) endpoint which will run the ingest logic, and then change this endpoint to read from Postgres

Reply:
Agreed on this